entrypoint.sh

```diff
@@ -4,5 +4,10 @@ sed -i "s|\${bindPath}:|${SERVICE_PATH}:|g" /var/node_modules/serverless-python-
 sed -i -E "s/if \(options.dockerSsh\)/cmdOptions.push('-e', 'GIT_TOKEN=${GIT_TOKEN}'); if \(options.dockerSsh\)/g" /var/node_modules/serverless-python-requirements/lib/pip.js
 
 if [[ $@ = *"--no-publish"* ]]; then export NO_PUBLISH=true; else export NO_PUBLISH=false; fi
+if [[ ! $@ = *"--no-env"* ]] && [ ! -e /var/task/.env.yml ]; then
```

Comment:
@jonas-777 So when running the `remote` Docker compose service in order to deploy, the project directory is mounted inside the `remote` container (i.e. the serverless:latest container), so this means your `/var/task/.env.yml` will be persisted even after the deployment. For what you're trying to do, I think you should delete `/var/task/.env.yml` before you start calculating the environment.